source/server/getting-started/installation.rst

```diff
@@ -45,18 +45,38 @@ Dedicated Servers
     If you already have a Forge server, just put the Sponge mod into your ``mods`` folder. Remember to update your Forge
     version if it's older than the one that :doc:`sponge requires <implementations/spongeforge>`.
 
-Via GUI
---------
+Installing Forge via Commandline
+--------------------------------
+
+1. Visit the `Minecraft Forge website <http://files.minecraftforge.net/>`_ and click "Show all downloads" to view the full
+   set of available options. Identify the latest version, or any version that's newer than the one listed :doc:`in the
```

Comment:
That bit about "newer than" is copypasted from the existing content. I'll update the page.